tiatoolbox/wsicore/wsireader.py

```diff
@@ -1171,15 +1342,37 @@ def read_bounds(
         interpolation="optimise",
         pad_mode="constant",
         pad_constant_values=0,
+        location_is_at_requested=False,
```

Comment:
I think this needs to be added to the parent class docstring for this function and the base class function signature updated. I think we could probably remove the word 'is' here too, e.g. `location_at_requested`.

Reply:
Its already there. Unless you are referring to other place?
https://github.com/TIA-Lab/tiatoolbox/blob/6e24aec1c117736a01abc6e6b2e31defa56a2279/tiatoolbox/wsicore/wsireader.py#L712
https://github.com/TIA-Lab/tiatoolbox/blob/6e24aec1c117736a01abc6e6b2e31defa56a2279/tiatoolbox/wsicore/wsireader.py#L756

Reply:
I agree to change it to `location_at_requested`